depends/packages/azmq.mk

```diff
@@ -0,0 +1,25 @@
+package=azmq
+$(package)_version=1.0
+$(package)_download_path=https://github.com/zeromq/$(package)/archive/
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+$(package)_download_file=v$($(package)_version).tar.gz
+$(package)_sha256_hash=c204c731bcb7810ca3a2c5515e88974ef2ff8d0589e60a897dc238b369180e7b
```

Comment:
latest version?

Reply:
Looks like it, although there have been numerous commits to master since then.